firefox-ios/Client/Frontend/TrackingProtection/CertificatesViewController.swift

```diff
@@ -0,0 +1,276 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Common
+import Shared
+import SiteImageView
+
+import Security
+import CryptoKit
+import X509
+import SwiftASN1
+
+struct CertificateKeys {
+    static let commonName = "CN"
+    static let country = "C"
+    static let organization = "O"
+}
+
+struct CertificatesUX {
+    static let sectionLabelWidth = 150.0
+    static let sectionLabelMargin = 20.0
+    static let sectionItemsSpacing = 40.0
+    static let allSectionItemsSpacing = 10.0
+    static let allSectionItemsTopMargin = 20.0
+    static let headerStackViewSpacing = 16.0
+    static let titleLabelMargin = 8.0
+    static let titleLabelTopMargin = 20.0
+    static let headerStackViewMargin = 8.0
+    static let headerStackViewTopMargin = 20.0
+    static let tableViewSpacerTopMargin = 20.0
+    static let tableViewSpacerHeight = 1.0
+    static let tableViewTopMargin = 20.0
+}
+
+typealias CertificateItems = [(key: String, value: [String])]
```

Comment:
As far as I can tell, `value` is never actually an array. So `value` can just be `String` (involves some refactoring). That'll also make `tableView:cellForRowAt` a bit easier to read.

Reply:
If you take a look in getDNSNames method, from viewModel, there is a case where we need a value as an array because for DNSName key we have a list of values.